pkg/minikube/assets/addons.go

```diff
@@ -742,6 +742,13 @@ var Addons = map[string]*Addon{
 		map[string]string{
 			"Headlamp": "ghcr.io",
 		}),
+	"cloud-spanner": NewAddon([]*BinAsset{
```

Comment:
Thank you for this PR, lets choose more descriptive name that makes sure it is talking about Google spanner

Reply:
Thanks Medya for suggestion. Do you mean adding google prefix everywhere? Also, cloud-spanner is itself a unique name. So, instead of convoluting it with google prefix everywhere, I wonder if we just made it explicit in the documentation.